content/brazilian-portuguese/android-tictactoe/printconsole.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Imprimindo no console"
+date: 2021-03-13T14:17:07.42-07:00
+draft: false
+weight: 4
+---
+Às vezes, o jogo ou aplicativo pode não funcionar da maneira esperada ou você deseja gravar algo no console para fins de registro. Fora da depuração do aplicativo, você pode usar `System.out.println()` para escrever sua mensagem. Procure suas mensagens na janela "logcat" na parte inferior do Android Studio:
+<img src="../resources/_gen/images/println.gif" height="60%" width="60%" title="System.out.println()" alt="Janela Logcat no Android Studio mostrando mensagens criadas usando a função de registro System.out.printLn"/>
+
+{{% notice tip %}}
+
+## Trabalhando juntos
+
+1. Sem remover as aspas, tente alterar `"Reiniciando o jogo Tac-Tac-Toe!"` para `"Reiniciando o jogo Tic-Tac-Toe!"` ou qualquer outra frase que desejar. Pressione executar para ver se alguma coisa muda.
```

Comment:
Translate "Tic-Tac-Toe"
@ozhang22 the proposal "try changing `"Restarting the Tac-Tic-Toe Game!"` to `"Restarting the Tic-Tac-Toe Game!"` looks weird, as it changes nothing. Is that right?